prowler/providers/aws/services/organizations/organizations_opt_out_ai_services_policy/organizations_opt_out_ai_services_policy.metadata.json

```diff
@@ -1,14 +1,14 @@
 {
   "Provider": "aws",
   "CheckID": "organizations_opt_out_ai_services_policy",
-  "CheckTitle": "Ensure that AWS Organizations opt-out of AI services policy is enabled.",
+  "CheckTitle": "Ensure that AWS Organizations opt-out of AI services policy is enabled and disallow child-accounts to overwrite this policy.",
   "CheckType": [],
   "ServiceName": "organizations",
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service::account-id:organization/organization-id",
   "Severity": "low",
   "ResourceType": "Other",
-  "Description": "This control checks whether the AWS Organizations opt-out of AI services policy is enabled. The control fails if the policy is not enabled.",
+  "Description": "This control checks whether the AWS Organizations opt-out of AI services policy is enabled and whether child-accounts are disallowed to overwrite this policy. The control fails if the policy is not enabled or if child-accounts are not disallowed to overwrite this policy.",
   "Risk": "By default, AWS may be using your data to train its AI models. This may include data from your AWS CloudTrail logs, AWS Config rules, and AWS GuardDuty findings. If you opt out of AI services, AWS will not use your data to train its AI models.",
   "RelatedUrl": "https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_ai-opt-out_all.html",
   "Remediation": {
```

Comment:
Can you update the remediation code, please?

Reply:
Following the existing remediation code, it would be extended to:
```
aws organizations enable-policy-type --root-id <root-id> --policy-type AI_SERVICES_OPT_OUT {'services': {'default': {'opt_out_policy': {'@@operators_allowed_for_child_policies': ['@@none'], '@@assign': 'optOut'}}}}
```
However, I don't think that it's possible to provide the policy content by the command `enable-policy-type`. At least, I don't see it in the AWS documentation. Feel free to proof me wrong please, I'd be happy.
I rather think that you first need to enable the policy type by using the command `enable-policy-type` as follows:
```
aws organizations enable-policy-type \
    --root-id <root-id> \
    --policy-type AI_SERVICES_OPT_OUT
```
Then you have to create the policy:
```
aws organizations create-policy \
    --content '{
        "services": {
            "@@operators_allowed_for_child_policies": ["@@none"],
            "default": {
                "@@operators_allowed_for_child_policies": ["@@none"],
                "opt_out_policy": {
                    "@@operators_allowed_for_child_policies": ["@@none"],
                    "@@assign": "optOut"
                }
            }
        }
    }' \
    --name "AIOptOutPolicy" \
    --type "AI_SERVICES_OPT_OUT" \
    --description "Policy to ensure opt-out for all AI services and prevent child accounts from overriding it."
```
Last but not least you have to attach the policy:
```
aws organizations attach-policy \
    --policy-id <policy-id> \
    --target-id <root-id>
```
I don't know how you could add all this to prowler's remediation code section.

Reply:
You are right! I will remove the actual remediation code since we cannot put more than one command there at the moment.